### ISSUE TITLE
Problems when executing from external drive directory

### ISSUE THREAD

**Issue by Songpyeon** [instaloader/instaloader#671]
I downloaded this
https://github.com/instaloader/instaloader/releases/tag/v4.4.4rc1
and now I'm having some errors:

```
C:\Users\USER>pushd Z:\
Z:\>instaloader igtest111
Argument List: ['C:\\Users\\USER\\AppData\\Local\\Programs\\Python\\Python37-32\\Scripts\\instaloader-script.py', 'igtest111']
Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python37-32\Scripts\instaloader-script.py", line 13, in <module>
    load_entry_point('instaloader==4.4.4rc1', 'console_scripts', 'instaloader')()
  File "c:\users\user\appdata\local\programs\python\python37-32\lib\site-packages\instaloader\__main__.py", line 239, in main
    parser = ArgumentParser(description=__doc__, add_help=False, usage=usage_string(),
  File "c:\users\user\appdata\local\programs\python\python37-32\lib\site-packages\instaloader\__main__.py", line 32, in usage_string
    argv0 = get_simple_argv0()
  File "c:\users\user\appdata\local\programs\python\python37-32\lib\site-packages\instaloader\__main__.py", line 23, in get_simple_argv0
    dirname = os.path.relpath(os.path.dirname(sys.argv[0]))
  File "c:\users\user\appdata\local\programs\python\python37-32\lib\ntpath.py", line 562, in relpath
    path_drive, start_drive))
ValueError: path is on mount 'C:', start on mount 'Z:
```

Everything works fine if I'm not changing the execution directory, but I want files to be saved on another drive. Is there a way to specify output directory somehow?

**Comment by aandergr** [comment]
Thanks for trying out our current pre-release and for reporting this bug! It seems to be introduced with 1c13d5a30b09f9687f98f6bcc0e46cc4b57c29a2, where (besides switching to Powershell as default shell to be started if instaloader.exe is called from within the file explorer) we intended to print a nice relative path for "instaloader.exe" in Instaloader's brief usage output.

As a workaround until it is fixed, you can start Instaloader with a current working directory on the same drive where the exe file resides, and pass the [--dirname-pattern flag](https://instaloader.github.io/basic-usage.html#filename-specification) to Instaloader, i.e.
```
C:\Users\USER> instaloader --dirname-pattern="C:\{target}"
```

(it may be that "C:\\{target}" (double backslash) is required instead of "C:\{target}", I am not sure.)

**Comment by Songpyeon** [comment]
Yeah the workaround works pretty nice, both single/double backslash seems to be working, but when I have .bat file on another drive than C: (where instaloader is installed) I have to use **pushd** and it works just as fine.

> pushd C:
> instaloader igname --dirname-pattern="Z:\\{target}"

Thanks for help!

**Comment by aandergr** [comment]
Fixed with [Version 4.4.4rc2](https://github.com/instaloader/instaloader/releases/tag/v4.4.4rc2).
